NuCivic/dkan
# Fix access to additional_settings tabs for content creators CIVIC-6181

Issue: CIVIC-6181 & CIVIC-6182

## Description

**CIVIC-6181:** Content creators can modify authoring information if members of group

**CIVIC-6182:** No publishing options if content creator doesn't belong to a group

## How to Reproduce CIVIC-6181
- create a Content Creator user and assign the user to a group
- switch to the user
- create or edit a dataset or resource
- check to see if the *Authoring information* tab fields are displayed and editable

## How to Reproduce CIVIC-6182
- create a Content Creator user and do *not* assign them to a group
- create or edit a dataset and you will *not* see the *Publishing options* tab fields

## Acceptance Criteria

- Content Creators are not able to modify the *Authoring information* in datasets or resources when they are assigned to a group
- Content Creators that aren't assigned to groups are able to save draft content

## Reminders
